gbpservice/nfp/orchestrator/config_drivers/heat_driver.py

```diff
@@ -200,13 +210,44 @@ def keystone(self, user, pwd, tenant_name, tenant_id=None):
             return self.keystoneclient.get_scoped_keystone_token(
                 user, pwd, tenant_name)
 
+    def _get_heat_client_v1(self, tenant_id, assign_admin=False):
+        if assign_admin:
+            try:
+                self._assign_admin_user_to_project(tenant_id)
```

Comment:
Why we need new function ?
Except assign_admin variable, code is same with function _get_heat_client()